### DISCUSSION TITLE
Citing Cognition.Run

### DISCUSSION THREAD

**Discussion by BobRoss048** [jspsych/jsPsych#1747] [Q&A]
Hello,

I know this forum is not _specifically_ for Cognition.Run, but I was just curious if anyone knows or has an example of how to cite using Cognition.Run in-text in 7th edition APA format.

I currently have it (Cognition, 2021), but I wanted to make sure that's accurate and respect the site.  

Thank you for your time.

**Comment by DanielBruijn** [comment]
Hey Bob,

I have this question as well. I think, but i am not sure, that just referencing jsPsych is good enough. You can say something like 'i used jsPsych v6.3.1 (De Leeuw, 2015) in cognition.run.' 
And then in reference: 
de Leeuw, J. R. (2015). jsPsych: A JavaScript library for creating behavioral experiments in a web browser. Behavior Research Methods, 47(1), 1-12. doi:10.3758/s13428-014-0458-y.

(gotten from: https://www.jspsych.org/about/about/)

-Daniel

**Reply by jodeleeuw** [reply]
I think it would be great to cite cognition.run in addition to jspsych. @javidalpe do you have a preferred citation for folks?

**Reply by DanielBruijn** [reply]
Good to know, i said so because i couldn't find anything on the website.
